rfcs/054-prismic-api/README.md

```diff
@@ -0,0 +1,110 @@
+- The API url structure should also be consistent with what appears on the front end of the site. For example, an article on the site appears at `/articles/{id}`, so the API equivalent should be `/articles/{id}`. The same should be true for search results.
+- The new API service should be written in typescript, following patterns set by the [concepts API](../050-concepts-api/README.md) for filtering, pagination, error handling, etc.
+- We shouldn't create new IDs for exhibitions, events, stories, etc. There aren't any situations where we should need to merge content from multiple sources, so we should use the document IDs directly from prismic.
+- The elasticsearch index mapping should represent the contract between the pipeline and the API. The API shouldn't need to know anything about the structure of the data in prismic.
+
+## Proposed endpoints
+
+### `/articles`
+
+Filter for type eg stories, webcomics
+
+#### Different kinds of thing (are these 'labels'?)
+
+- story <https://wellcomecollection.org/articles/Y9vktREAACYA1v2U> at /articles
+- webcomic <https://wellcomecollection.org/articles/Y86rshEAACcAI4g4> at /articles
+- book <https://wellcomecollection.org/books/Y7QWrREAAHC43oH0> at /books
+- series <https://wellcomecollection.org/series/WfpEBCoAACrdd43y> composed of articles
+- in pictures <https://wellcomecollection.org/articles/Y-oAIhAAACAAix32> at /articles
+
+#### Filters
+
+- date
+- contributor
+- series
+- contentType
+
+#### Sort
+
+- date
+- relevance
+
+### `/exhibitions`
+
+#### Filter
+
+- date
+- status
+- location
+- contributor?
+- format (ie permanent, exhibition, installation)
+- season
+
+#### Sort
+
+- date
+- relevance
+
+### `/events`
+
+#### Filter
+
+- date
+- accessibility status (is it audio described, signed, etc)
+- location
+- format (eg discussion)
+- event-series
+- season
+- audience (eg schools)
+- contributor
+- isOnline?
+- availableOnline?
+
+#### Sort
+
+- date
+- relevance
+
+### `/books`
+
+?????????
+
+## Open questions
+
+- How do we represent functional content?
+- Should we include the full prismic API response at eg `/articles/{id}`?
+    - should we be able to list all of the docs of a given type at once? what should that url look like?
+- If we want to avoid maintaining multiple versions of the same code, should this work be done as an extension to the concepts API?
+- Works exist at [api.wellcomecollection.org/catalogue/v2/works](api.wellcomecollection.org/catalogue/v2/works), concepts at [/catalogue/v2/concepts](api.wellcomecollection.org/catalogue/v2/concepts), images at [/catalogue/v2/images](api.wellcomecollection.org/catalogue/v2/images).  
+Stories etc aren't part of the catalogue - should the URL be different?
+- what are the top level types? is it articles? stories? series?
+- where do books go? they appear on the stories landing page, so should they appear in stories search results? or are they at a separate endpoint?
+- what is a serial? what is a series?
```

Comment:
@dannybirchall @davidpmccormick @gestchild @jtweed 

We'd like to get this one clarified, I think there was some legacy naming in the codebase here and we'd like to be using the correct language from the start.

Reply:
I believe a serial is a particular type of series, i.e. one with a schedule which is a list of titles it contains and their corresponding publish date

Reply:
That's right! Thank you.